docs/components/popover.md

```diff
@@ -330,11 +330,10 @@ This is the slot that takes in the content of the popover.
 
 ### Events / Loading Content
 
-- `opened` - emitted once the popover has been opened
-- `closed` - emitted when the popover has been triggered closed (emits on all
-triggers)
+- `opened` - emitted once the popover has been opened, also passes in whether or not the popover is currently being displayed
+- `closed` - emitted when the popover has been triggered closed (emits on all triggers)
 
-<KPop @opened="loadSomething" @closed="onClose">
+<KPop @opened="(isShow) => loadSomething(isShow)" @closed="onClose">
```

Comment:
is `isShow` a common Vue name?  I've seen `isOpen`/`open` and `isVisible`/`visible` used a lot more.  that's not even a nit (we can call it whatever you think is best), but rather just a curious question.

Reply:
right, maybe I should use `isOpen`

Reply:
I'm completely cool with whatever, was just curious :smile: